utils/xprof_utils.hpp

```diff
@@ -11,7 +11,8 @@ enum backend_e{ BACKEND_UNKNOWN = 0,
                 BACKEND_CUDA = 3,
                 BACKEND_OMP_TARGET_OPERATIONS = 4,
                 BACKEND_OMP = 5,
-                BACKEND_HIP = 6 };
+                BACKEND_HIP = 6,
+                BACKEND_MAX = 7 };
 
 constexpr int backend_level[] = { 2, 2, 2, 2, 1, 0, 2 };
```

Comment:
I think we should remove this one. Nobody is using backend_level except tally.cpp no?

Reply:
So do you want me to move it to tally?, but this is the default